src/Subsites/SubsiteSecurityReport.php

```diff
@@ -12,13 +12,4 @@
 class SubsiteSecurityReport extends Extension
 {
 
-    /**
-     * Columns in the report
-     *
-     * @var array
-     * @config
-     */
-    private static $columns = array(
```

Comment:
Should this be removed, if we're only [deprecating the extension](https://github.com/silverstripe/silverstripe-securityreport/pull/41/files) at this stage?

Reply:
It’s moved into YAML instead

Reply:
Right, and this would be merged up so the deprecation notice applies to a now empty extension - sweet got it 👍